components/button/index.tsx

```diff
@@ -92,6 +93,7 @@ export interface ButtonProps {
     style?: React.CSSProperties
     disabled?: boolean
     className?: string
+    tabindex?: number
```

Comment:
Use `tabIndex` to be consistent with react
and also
https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_focus_03